fairscale/utils/reduce_scatter_bucketer.py

```diff
@@ -26,9 +26,15 @@ def flush(self) -> None:
             assert len(self.callbacks) == 0
             return
         # reduce-scatter bucket
-        dist.reduce_scatter(
-            self.output_shard[: self.offset], list(self.data[:, : self.offset].unbind(0)), group=self.group
-        )
+        if hasattr(dist, "_reduce_scatter_base"):
+            # ignore
+            dist._reduce_scatter_base(  # type: ignore
```

Comment:
Save as above, can you add the typing info to the stubs dir? If you prefer me to add that's totally fine. I can do them in a follow up PR.

Reply:
Thanks! That would be great!